common/app/common/Logback/Logstash.scala

```diff
@@ -0,0 +1,37 @@
+package common.Logback
+
+import conf.switches.Switches
+
+import play.api.{Logger => PlayLogger, Application => PlayApp, GlobalSettings, LoggerLike}
+import conf.Configuration
+
+
+case class LogStashConf(stream: String, region: String, role: String)
+
+trait Logstash extends GlobalSettings {
+
+  override def onStart(app: PlayApp) = {
+    super.onStart(app)
+    Logstash.init(PlayLogger) // Setup Play default Logger to send logs to logstash
+  }
+}
+
+object Logstash {
+
+  val config = for {
+    stream <- Configuration.Logstash.stream
+    region <- Configuration.Logstash.streamRegion
+    role <- Configuration.Logstash.streamRole
+  } yield {
+    LogStashConf(stream, region, role)
+  }
+
+  def init(logger: LoggerLike) = {
+    if(Switches.LogstashLogging.isSwitchedOn) {
```

Comment:
Is it possible to have this switch default to off when running in local dev mode?